es6-modules/src/components/todo-list.js

```diff
@@ -0,0 +1,52 @@
+import createElement from '../lib/create-element';
```

Comment:
While this works it is better to import this from the `lib/index.js` file. For example:
```JAVASCRIPT
import { createElement } from '../lib';
```